fedmsg_meta_fedora_infrastructure/tests/pagure.py

```diff
@@ -1324,6 +1324,237 @@ class TestNewPullRequestNew(Base):
     }
 
 
+class TestPullRequestFlagAdded(Base):
+    """ These messages are published when a someone flags a pull-request
+    on a project on `pagure <https://pagure.io>`_.
+    """
+    expected_title = "pagure.pull-request.flag.added"
+    expected_subti = 'Jenkins flagged the pull-request#1: "Tests failed"'
```

Comment:
Maybe change this to ``Jenkins flagged test #1 with "Tests failed"``?

No need to say it is a pull-requests -- they will all be pull requests.  Plus, it might be nice to have the repo name in there.